src/Workspaces/Core/Portable/FindSymbols/FindReferences/FindReferencesSearchEngine_FindReferencesInDocuments.cs

```diff
@@ -112,7 +112,7 @@ async ValueTask PerformSearchInProjectAsync(ImmutableArray<ISymbol> symbols, Pro
                 foreach (var (_, location) in referencesForFinder)
                 {
                     var group = await ReportGroupAsync(symbol, cancellationToken).ConfigureAwait(false);
-                    await _progress.OnReferenceFoundAsync(group, symbol, location, cancellationToken).ConfigureAwait(false);
+                    await _progress.OnReferencesFoundAsync([(group, symbol, location)], cancellationToken).ConfigureAwait(false);
```

Comment:
>OnReferencesFoundAsync [](http://example.com/codeflow?start=36&length=22)

Could the OnReferencesFoundAsync calls be batched here and below?

Reply:
Done.